fastplotlib/graphics/selectors/_linear_region.py

```diff
@@ -109,13 +116,21 @@ def _feature_changed(self, key: Union[int, slice, Tuple[slice]], new_data: Any):
             selected_ixs = None
             selected_data = None
 
+        # get pygfx event and reset it
+        pygfx_ev = self._parent._pygfx_event
+        self._parent._pygfx_event = None
+
         pick_info = {
             "index": None,
             "collection-index": self._collection_index,
             "world_object": self._parent.world_object,
             "new_data": new_data,
             "selected_indices": selected_ixs,
-            "selected_data": selected_data
+            "selected_data": selected_data,
+            "graphic": self._parent,
+            "delta": self._parent.delta,
+            "pygfx_event": pygfx_ev,
+            "move_info": self._parent._move_info
         }
 
         event_data = FeatureEvent(type="bounds", pick_info=pick_info)
```

Comment:
So in other `GraphicFeatures` the `type` is things like `"colors"`, `"data"` etc. 

But here, for `type` do you think we should:

1. Use the same name for all `Selector` graphics, just `type="selector"`?
2. Have type specificy the subclass, i.e. `"linear-selector"`, `"linear-region-selector"`, etc.

(2) might be a bit redundant because we can just use `type(event.pick_info["graphic"])` to get the actual type of the Selector. I see (1) being useful because all `Selectors` behave similarly, they have a `get_selected_index()` or `get_selected_indices()` method etc. So `Selectors` _should_ work with `Graphic.link()` with a little bit of work and this might be easier with the `type` as `"selector"` instead of the specific subtypes which we then parse out.

You wrote the `link()` system so let me know your thoughts! :D

Reply:
I would say (1) is better, no need to over complicate things when they behave so similarly anyway